app/views/change_requests/all.slim

```diff
@@ -4,7 +4,7 @@ div.content-container
     div.center
       a.archived-option href="#{all_change_requests_path}?archived=true" tabindex="-1"
         button.btn.btn-primary Include Archived Requests
-  table.clean-table.responsive id=(@change_requests.empty? ? nil : "changeRequestsAllTable")
+  table.clean-table.review-table.responsive id=(@change_requests.empty? ? nil : "changeRequestsAllTable")
```

Comment:
This is a change request table, not a review-table, but since both .review-table AND changeRequestsAllTable get the DataTable feature applied it was trying to doubly implement the change and thus the error